local-cli/cli.js

```diff
@@ -7,21 +7,10 @@
  * @format
  */
 
-'use strict';
-
-// gracefulify() has to be called before anything else runs
-require('graceful-fs').gracefulify(require('fs'));
-
-// This file must be able to run in node 0.12 without babel so we can show that
-// it is not supported. This is why the rest of the cli code is in `cliEntry.js`.
-require('./server/checkNodeVersion')();
-
-require('../setupBabel')();
-
-var cliEntry = require('./cliEntry');
+var cli = require('react-native-local-cli');
```

Comment:
(nit): why change the name of the variable ?

Reply:
I ended up overwriting this file with a local copy I created upfront. Not intentional, I can revert if needed.